CorsixTH/Lua/objects/plant.lua

```diff
@@ -73,9 +73,6 @@ object.orientations = {
 -- For plant: droop down: 1950
 -- back up again: 1952
 
--- The states specify which frame to show
-local states = {"healthy", "drooping1", "drooping2", "dying", "dead"}
```

Comment:
This is currently the only documentation for ```current_state```, best to replace it with a comment indicating what each of the current_state values mean.